.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+on: [push]
+jobs:
+  build:
+    name: Build, lint, and test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - name: Install deps
+        run: yarn install
```

Comment:
Should we use a version with caching?

Reply:
We should, this was me debugging something!